### PR TITLE
removes map when vw < 600px

### PR DESCRIPTION
**Pull request by smalex-z**
## Describe your changes

Hides the embedded Google Maps on the GM page when VW < 600px. This will keep the page responsive and prevent stretching.

## Issue ticket number and link
Fixes Issue #721 

## Checklist before requesting a review
- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by snigdha-kansal — COMMENTED** [review]
Ideally, we still want users to be able to see the map on mobile view.
A better solution would be to make the display responsive, if possible.

**Comment by snigdha-kansal** [comment]
Can we maintain the use of single quotes for consistency? Thanks

**Comment by smalex-z** [comment]
> Can we maintain the use of single quotes for consistency? Thanks

done

File: pages/gm.js [hunk 1]
```diff
@@ -218,9 +218,16 @@ function gm() {
 			<div className='content-container-tight text-center'>
         <div id='info-wrapper'>
           <h2>Relevant information</h2>
-          <div className='flex'>
-            <div>
-              <iframe src='https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d13219.85897956824!2d-118.4441451!3d34.070418!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x62f97fe423993f80!2sUCLA%20Ackerman%20Union!5e0!3m2!1sen!2sin!4v1663063464157!5m2!1sen!2sin' width='450' height='400' style={{border: 0}} allowfullscreen='' loading='lazy' title='google maps embed of ackerman grand ballroom' referrerPolicy='no-referrer-when-downgrade'></iframe>
+          <div className='flex map-container'>
+            <div style={{flexGrow: 1, width: '100%'}}>
+              <iframe
+                src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d13219.85897956824!2d-118.4441451!3d34.070418!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x62f97fe423993f80!2sUCLA%20Ackerman%20Union!5e0!3m2!1sen!2sin!4v1663063464157!5m2!1sen!2sin"
```
Comment (snigdha-kansal) [inline pages/gm.js:224]: " -> ' for this one too!

File: pages/gm.js [hunk 2]
```diff
@@ -218,9 +218,16 @@ function gm() {
 			<div className='content-container-tight text-center'>
         <div id='info-wrapper'>
           <h2>Relevant information</h2>
-          <div className='flex'>
-            <div>
-              <iframe src='https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d13219.85897956824!2d-118.4441451!3d34.070418!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x62f97fe423993f80!2sUCLA%20Ackerman%20Union!5e0!3m2!1sen!2sin!4v1663063464157!5m2!1sen!2sin' width='450' height='400' style={{border: 0}} allowfullscreen='' loading='lazy' title='google maps embed of ackerman grand ballroom' referrerPolicy='no-referrer-when-downgrade'></iframe>
+          <div className='flex map-container'>
+            <div style={{flexGrow: 1, width: '100%'}}>
```
Comment (snigdha-kansal) [inline pages/gm.js:222]: being nit picky but can we create a class for this and use the classname instead? thanks

**Review by yeadamk — APPROVED** [review]
lgtm

**Pull request merged**